src/JavaScriptSDK.Interfaces/ITelemetryItem.ts

```diff
@@ -8,22 +8,22 @@
 export interface ITelemetryItem {
     /**
      * Unique name of the telemetry item
-     */ 
+     */
     name: string;
 
     /**
      * Timestamp when item was sent
-     */ 
+     */
     timestamp?: Date;
```

Comment:
We called this property "time", can you verify what is the expected property name?